docs/model_zoo/action.rst

```diff
@@ -0,0 +1,62 @@
+.. _gluoncv-model-zoo-action:
+
+Action
+====================
+
+.. role:: greytag
+
+Action Recognition
+~~~~~~~~~~~~~~~~~~~~~
+
+Table of pre-trained models for video action recognition and their performance.
+
+.. hint::
+
+  Training commands work with this script:
```

Comment:
space all lines under `hint` correctly. otherwise it may be rendered unexpectedly.

Reply:
Thank you, align them now.